docs/index.rst

```diff
@@ -1,33 +0,0 @@
-.. NI Modular Instruments Python API documentation master file, created by
-   sphinx-quickstart on Fri Jul 14 13:04:36 2017.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
-NI Modular Instruments Python Documentation Redirect
-============================================================
-
-Documentation for versions 1.4.4 and earlier of the following modules can be found here:
-
-* nidcpower
-* nidigital
-* nidmm
-* nifgen
-* nimodinst
-* niscope
-* niswitch
-* nise
-* nitclk
-
-Simply select the version you wish to view documentation for.
-
-For newer documentation, refer to the individual **Read the Docs** projects.
-
-* `nidcpower <https://nidcpower.readthedocs.io/en/latest>`_
```

Comment:
I think I should update this to stable and manually rebuild one last time, before I delete.

Reply:
Hmm, it may be too late to update the link for the stable version and version 1.4.5 pages.

Reply:
Reverted and updated the links.
I actually already deleted the webhook for the old project. I'll probably need to:
1. Recreate the webhook
2. Delete the version tag from the 1.4.5 release
3. Create a 1.4.5 tag for this PR Commit
4. Delete the webhook after the old project documentation builds (should rebuild 1.4.5, stable and latest to list links to new stable docs)
5. Delete the 1.4.5 tag
6. Readd the 1.4.5 tag to the 1.4.5 release

Reply:
This is done.